fix: Change order consignment outlet position from REPLACE to AFTER

surya-prakash-singh1 opened the pull request:
ix: Change order consignment outlet position from REPLACE to AFTER

JedidiahAwuku committed:
fix: Change order consignment outlet position from REPLACE to AFTER (#17119)

surya-prakash-singh1 committed:
Merge branch 'release/6.0.x' into fix/CXSPA-2928

surya-prakash-singh1 committed:
Merge branch 'release/6.0.x' into fix/CXSPA-2928

surya-prakash-singh1 committed:
Merge branch 'release/6.0.x' into fix/CXSPA-2928

JedidiahAwuku committed:
Merge branch 'release/6.0.x' into fix/CXSPA-2928

JedidiahAwuku committed:
fix: failing pipeline tests

Remove unused lifecycle hook from PickupInStoreOrderConsignmentComponent

JedidiahAwuku committed:
Merge branch 'release/6.0.x' into fix/CXSPA-2928

JedidiahAwuku committed:
Merge branch 'release/6.0.x' into fix/CXSPA-2928

JedidiahAwuku committed:
Merge branch 'release/6.0.x' into fix/CXSPA-2928

WeizhengSap commented on package-lock.json:
```diff
@@ -20647,7 +20647,7 @@
     "projects/eslint": {
       "name": "@spartacus/eslint-plugin-eslint",
-      "version": "0.0.0-dev",
+      "version": "6.0.0-1",
```
why we need change this?

JedidiahAwuku replied:
Removed. thanks

WeizhengSap commented on feature-libs/order/styles/components/order-details/_order-details-items.scss:
```diff
@@ -35,12 +35,6 @@
     padding: 0 30px;
   }
 
-  .cx-review-header {
```
I think we need this. please double check

JedidiahAwuku replied:
Hi Wei, it seems styling for the `cx-review-header` has been already made in the `_review.scss` file or is this deliberate? The UI looks the same when added and removed.

JedidiahAwuku committed:
fix: address comments from review

Remove unwanted version from projects/eslint in package-lock.json

Add style for cx-review-header in order details item component

JedidiahAwuku committed:
Merge branch 'release/6.0.x' into fix/CXSPA-2928

JedidiahAwuku committed:
fix: remove unwanted border from cx-review-header

JedidiahAwuku committed:
Merge branch 'fix/CXSPA-2928' of https://github.com/SAP/spartacus into fix/CXSPA-2928
